corehq/apps/userreports/tests/test_report_filters.py

```diff
@@ -215,24 +215,26 @@ def test_string_datatype(self):
         self.filter_spec['datatype'] = 'string'
         filter = ReportFilterFactory.from_spec(self.filter_spec)
         tests = (
-            (1, '1'),
-            (1.2, '1.2'),
+            ('1', '1'),
+            ('1.2', '1.2'),
```

Comment:
Since [`filter.value()`](https://github.com/dimagi/commcare-hq/blob/3a996f234f7eb7e28fbacec5dde5bd48f0e32a3c/corehq/apps/reports_core/filters.py#L231) is used to transform the value passed in the request by the user to the appropriate python type, I believe that the input to `filter.value()` should always be a string. Does that sound right to you @czue , or did you have another use case in mind with these tests?